lib/chains/elementChain.js

```diff
@@ -292,6 +295,10 @@ const getComposers = (data) => {
 		}
 	}
 
+	if (!data.comparator && data.selector && data.selector.video && !data.isNegated) {
```

Comment:
Confirmed just now that in UI Test Editor we can check video properties on any element, not only on video shortcut. So, `isPlaying`, `isPaused` and `isStopped` should be available on any element, not just the one defined as `video` selector.